src/NuGet.Core/NuGet.Commands/PublicAPI/net472/PublicAPI.Unshipped.txt

```diff
@@ -1,2 +1,3 @@
 NuGet.Commands.SignArgs.PackagePaths.get -> System.Collections.Generic.IReadOnlyList<string>
 NuGet.Commands.SignArgs.PackagePaths.set -> void
+static NuGet.Commands.MSBuildProjectFrameworkUtility.GetProjectFramework(string projectFilePath, string targetFrameworkMoniker, string targetPlatformMoniker, string targetPlatformMinVersion, string clrSupport) -> NuGet.Frameworks.NuGetFramework
```

Comment:
I'm starting to feel like we should change the public API to use `Dictionary<string, string>` rather than adding a new parameter every time a different project system wants to be yet another special case.

Reply:
Isn't creating a dictionary an overkill?
Prior to this change & .NET 5, those codepaths were consistent for quite a while. 

We'd merely be adding a dictionary to avoid making public API changes, not because it's considered good design.

Reply:
*shrug*  we also used a property bag in the nomination API to allow project system to pass us a bunch of properties, rather than creating a strongly typed object that represents every property we care about.